slp_mtmt/slp_mtmt/parse/mtmt_component_parser.py

```diff
@@ -5,14 +5,17 @@
 from slp_mtmt.slp_mtmt.parse.mtmt_trustzone_parser import MTMTTrustzoneParser
 from slp_mtmt.slp_mtmt.parse.resolvers.resolvers import get_type_resolver
 from slp_mtmt.slp_mtmt.util.border_parent_calculator import BorderParentCalculator
+from slp_mtmt.slp_mtmt.util.component_representation_calculator import ComponentRepresentationCalculator
 
 
 class MTMTComponentParser:
 
-    def __init__(self, source: MTMT, mapping: MTMTMapping, trustzone_parser: MTMTTrustzoneParser):
+    def __init__(self, source: MTMT, mapping: MTMTMapping, trustzone_parser: MTMTTrustzoneParser,
+                 diagram_representation: str):
         self.source = source
         self.mapping = mapping
         self.trustzoneParser = trustzone_parser
```

Comment:
Not exactly from this PR, but shouldn't this attribute's name be "trustzone_parser"? (also appears in mtmt_parser)